.docker/Dockerfile.meteor

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:focal
+
+ARG RELEASE
+
+ADD https://s3.amazonaws.com/com.meteor.static/packages-bootstrap/$RELEASE/meteor-bootstrap-os.linux.x86_64.tar.gz /
+
+RUN tar xzf /meteor-bootstrap-os.linux.x86_64.tar.gz
+
+FROM ubuntu:focal
+
+# Rocket.Chat build dependencies: g++ git build-essential (make + etc.) python
+RUN apt update && \
+    apt install --no-install-recommends -y \
+        g++ sudo git build-essential python ca-certificates && \
+    useradd -mG sudo meteor && \
+    echo '%meteor ALL=(ALL) NOPASSWD: ALL' > /etc/sudoers.d/meteor
+
+USER meteor
+WORKDIR /home/meteor
+
+COPY --from=0 /.meteor .meteor
```

Comment:
Is the `--from` here needed?  I thought only needed when reversing a different stage of a multi-stage dockerfile

Reply:
It is reversing the first stage. The downloaded archive isn't needed in the final image. 

The same *can* be achieved without the multi-stage way - but this just felt slightly more natural to me (also was my first thought).